generateResources.py

```diff
@@ -980,6 +982,31 @@ def item(filename_parts, *layers, parent='item/generated'):
 # LEATHER / HIDES
 blockstate(('placed_hide',), 'tfc:hide_rack', {})
 
+# AGRICULTURE
+
+for crop in CROPS.keys():
+    stages = CROPS[crop]['stages']
+    texture = CROPS[crop]['texture']
+    mymodel = CROPS[crop]['model']
+
+    variants = {'stage': {}}
+    for stage in range(stages):
+        variants['stage'][stage] = {
+            'textures': {
+                texture: "tfc:blocks/crop/%s_%d" % (crop, stage)
+            }
+        }
+    blockstate(
+        filename_parts=('crop', crop),
+        textures={texture: "tfc:blocks/crop/%s_0" % crop},
+        model=mymodel,
+        variants=variants
+    )
+    blockstate(
+        filename_parts=('dead_crop', crop),
+        textures={texture: "tfc:blocks/crop/%s_dead" % crop},
+        model=mymodel
+    )
```

Comment:
Python styling: don't separate arguments with newlines, don't name arguments unless they are necessary (i.e, non-positional), don't call things `mymodel`

Reply:
I used `mymodel` even though it's bad naming convention because when I used `model` it overloaded the function `model()` since it all runs on the import scope.